certbot/certbot/plugins/common.py

```diff
@@ -125,7 +123,7 @@ def auth_hint(self, failed_achalls):
                 .format(name=self.name, challs=challs))
 
 
-class Installer(Plugin):
+class Installer(AbstractInstaller, Plugin, metaclass=ABCMeta):  # pylint: disable=abstract-method
```

Comment:
Is `  # pylint: disable=abstract-method` here and above necessary? pylint seems to pass for me without this disable.

Reply:
Certainly a disable from an old version of the code which is not needed anymore.